azure-pipelines.yml

```diff
@@ -276,6 +292,23 @@ jobs:
       channels: # for ML.NET jobs we want to check .NET Core 3.1 and 5.0 only
         - main
 
+  # Ubuntu 1804 x64 F# benchmarks
```

Comment:
You should also add this and the other Windows section above in the Scheduled Private job section that starts on line 552. This way the tests will actually run in the lab and generate results. You only have them added to the public correctness section, which means they will only run as a part of PRs.

Reply:
Got it, added. I took some of the parameters from other jobs in that section, please check if it's correct.